backend/tests/legacy/test_api_snapshot.py

```diff
@@ -103,3 +103,30 @@ def test_allocations(
         for key, val in allocation.items():
             if key == "amount":
                 assert int(val) == 1000
+
+
+@pytest.mark.api
+def test_projects_basics(client: Client):
+    projects_epoch1, status_code = client.get_projects(1)
+    projects_number_epoch1 = len(projects_epoch1["projectsAddresses"])
+    assert projects_number_epoch1 > 0
+    assert projects_epoch1["projectsCid"], "projectsCid is empty"
+    assert status_code == 200
+    print("Projects epoch 1: ", projects_epoch1)
+
+    # forward time to the beginning of the epoch 2
+    move_to_next_epoch(2)
+    # wait for indexer to catch up
+    epoch_no = wait_for_sync(client, 2)
+    print(f"indexed epoch: {epoch_no}")
+
+    projects_epoch2, status_code = client.get_projects(2)
+    projects_number_epoch2 = len(projects_epoch2["projectsAddresses"])
+    assert projects_number_epoch2 > 0
+    assert projects_epoch2["projectsCid"], "projectsCid is empty"
+    assert status_code == 200
+    print("Projects epoch 2: ", projects_epoch2)
+
+    assert (
+        projects_number_epoch1 == projects_number_epoch2
+    ), "Number of projects is different in two epochs"
```

Comment:
Maybe adding a comment with a link to a design doc that explains this behavior would be nice? Do we have one?

Reply:
I believe on our api tests env we have a fixed list of projects for every epoch. This assertion is just to verify that 
a) The projects are there for 2nd epoch
b) The amount of projects for epoch 1 and 2 is the same
I am not sure if there is any design doc describing that. What do you suggest to add here to be more descriptive?

Reply:
https://github.com/golemfoundation/octant/blob/develop/contracts-v1/contracts/Proposals.sol#L84

This is the behavior I'm referencing. `If for epoch n+1 list of proposals is not set, check epoch n`.

Reply:
Is there a test case for that?